src/coreclr/nativeaot/docs/optimizing.md

```diff
@@ -32,10 +32,6 @@ Native AOT supports enabling and disabling all [documented framework library fea
 
```

Comment:
I have noticed that the "By default, the compiler tries to maximize compatibility ..." paragraph above is outdated. You can delete it while you are on it.